src/Core/Configurations/JsonConfigSchemaValidator.cs

```diff
@@ -71,8 +85,10 @@ public async Task<JsonSchemaValidationResult> ValidateJsonConfigWithSchemaAsync(
         {
             try
             {
-                JsonSchema jsonSchema = await JsonSchema.FromUrlAsync(runtimeConfig.Schema);
-                return jsonSchema.ToJson();
+                // Send a GET request to the URL specified in runtimeConfig.Schema to get the JSON schema.
+                HttpResponseMessage response = await _httpClient.GetAsync(runtimeConfig.Schema);
+                string jsonSchema = await response.Content.ReadAsStringAsync();
+                return jsonSchema;
```

Comment:
please include the error message that was appearing before this change to illustrate the URL connectivity was to blame and that there wasn't a problem with the Schema file that was downloaded by Newtonsoft jsonSchema

Reply:
created issue to track it #2124